src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/NomisData.kt

```diff
@@ -461,56 +461,20 @@ class NomisData(
         }
     }
 
-  // TODO - I chickened out of mapping AGENCY_LOCATIONS because AGY_LOC_ID is everywhere. Left it for the person who migrates agencies.
-  override fun agencyAddress(
+  override fun agencyLocation(
     agencyLocationId: String,
-    type: String?,
-    premise: String?,
-    street: String?,
-    locality: String?,
-    flat: String?,
```

Comment:
The top level nomis data builder now includes agency locations and agency addresses are its children.